mtags/src/main/scala/scala/meta/internal/mtags/SymbolOccurrenceOrdering.scala

```diff
@@ -19,18 +19,18 @@ object SymbolOccurrenceOrdering {
           if (byLine != 0) {
             byLine
           } else {
-            val byCharacter = Integer.compare(
-              a.startCharacter,
-              b.startCharacter
+            val byEnd = Integer.compare(
```

Comment:
We add the decorations comments at the end of a range, which is why we should first sort by end. We don't actually use the start character at all in SemanticdbSuite, where this is used.